CGPs/cgp-0046.md

```diff
@@ -0,0 +1,104 @@
+---
+cgp: 46
+title: cREAL oracle and constitution parameter activation
+date-created: 2021-11-01
+author: @martinvol
+status: DRAFT
+discussions-to: [cBRL forum proposal](https://forum.celo.org/t/cbrl-stable-asset/1281)
```

Comment:
Pending modifying forum post name which I'll do once I get the go ahead on the forum. See my comment there https://forum.celo.org/t/cbrl-stable-asset/1281/26?u=yaz

Once it modifies, we should modify it here too.